auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -248,159 +260,185 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
-    switch (true) {
-      case (this.isNameRequest(item.corpType.code) && !!item.nameRequest):
-        // Format name request state value for Display
-        if (!NrState[item.nameRequest?.state]) return 'Unknown'
-        return NrDisplayStates[NrState[item.nameRequest.state]] || 'Unknown'
-      case this.isTemporaryBusinessRegistration(item.corpType.code):
-        return BusinessState.DRAFT
-      case !!item.status:
-        return item.status.charAt(0)?.toUpperCase() + item.status?.slice(1)?.toLowerCase()
       default:
-        return BusinessState.ACTIVE
+        return '' // should never happen
     }
   }
 
-  /** Returns the folio number or a default message */
-  private folio (item: Business): string {
-    return item.nameRequest && (item.nameRequest.folioNumber || '')
-  }
-
-  /** Returns the last modified date string or a default message */
-  private lastModified (item: Business): string {
-    switch (true) {
-      case (!!item.lastModified):
-        return this.dateToPacificDate(new Date(item.lastModified))
-      case (!!item.modified):
-        return this.dateToPacificDate(new Date(item.modified))
-      default: return this.$t('notAvailable').toString()
+  /** Returns the type description. */
+  protected typeDescription (business: Business): string {
+    // if this is a name request then show legal type
+    if (this.isNameRequest(business)) {
+      const legalType: unknown = business.nameRequest.legalType
+      return GetCorpFullDescription(legalType as CorpTypeCd)
     }
+    // if this is an IA or registration then show legal type
+    if (this.isTemporaryBusiness(business)) {
+      const legalType: unknown = business.corpType.legalType
+      return GetCorpFullDescription(legalType as CorpTypeCd)
+    }
+    // else show nothing
+    return ''
   }
 
-  /** Returns the modified by value or a default message */
-  private modifiedBy (item: Business): string {
-    if (item.modifiedBy === 'None None' || !item.modifiedBy) {
-      return this.$t('notAvailable').toString()
-    } else { return item.modifiedBy || this.$t('notAvailable').toString() }
-  }
-
-  /** Redirect handler for Dashboard OPEN action */
-  private open (item: Business): void {
-    if (item.corpType.code === CorpType.NAME_REQUEST) {
+  /** Returns the status of the affiliation. */
+  protected status (business: Business): string {
+    if (this.isNameRequest(business)) {
+      // Format name request state value
+      const state = NrState[business.nameRequest.state]
+      if (!state) return 'Unknown'
+      else return NrDisplayStates[state] || 'Unknown'
+    }
+    if (this.isTemporaryBusiness(business)) {
+      return BusinessState.DRAFT
+    }
+    if (business.status) {
+      return business.status.charAt(0)?.toUpperCase() + business.status?.slice(1)?.toLowerCase()
+    }
+    return BusinessState.ACTIVE
+  }
+
+  /** Returns the last modified date string or a default message. */
+  // NOT USED AT THE MOMENT
```

Comment:
I dunno. We _might_ use this again so I didn't delete it, but I could be swayed. Thoughts?

Reply:
I think we should keep it. It isn't really cluttering up the code and it is really easily deleted later in the next refactor if it still isn't being used.